package.json

```diff
@@ -45,6 +46,10 @@
     "babel-preset-es2015": "^6.1.18",
     "eslint": "^1.10.1",
     "eslint-config-airbnb": "^1.0.0",
```

Comment:
You should remove `eslint-config-airbnb` if you installed `eslint-config-shakacode`

Reply:
Actually it was you)
Without this linting fails